<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleResetPage</class>
 <widget class="QWidget" name="VehicleResetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
font:24px  &quot;微软雅黑&quot;;
color:black;
border:2px solid white;
background-color:rgb(255,255,153);
}
QPushButton
{
	border-radius:8px;
font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 128, 128);border-top: 2px solid white;border-left: 2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="EBStateMP2lbl">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>150</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>DCU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EBStateMP1lbl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:RED;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>DCU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainNumM1lbl">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>140</y>
     <width>111</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255,255,153);
</string>
   </property>
   <property name="text">
    <string>09013</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateM2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>611</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DirectTC1lbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC1traingraphDirection.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>621</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="TrainNumMP2lbl">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>111</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255,255,153);
</string>
   </property>
   <property name="text">
    <string>09015</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainNumTC2lbl">
   <property name="geometry">
    <rect>
     <x>742</x>
     <y>140</y>
     <width>111</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255,255,153);

</string>
   </property>
   <property name="text">
    <string>09016</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EBStateM1lbl">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>150</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>DCU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainNumMP1lbl">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>140</y>
     <width>111</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255,255,153);

</string>
   </property>
   <property name="text">
    <string>09012</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>61</y>
     <width>841</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border:1px solid white;
border-radius:15px;
color:white;</string>
   </property>
   <property name="text">
    <string>DCU复位</string>
   </property>
  </widget>
  <widget class="QLabel" name="DirectTC2lbl">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/images/images/TC2traingraphDirection.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CabTC1lbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>31</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC1traingraphKeynoact.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainNumTC1lbl">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>140</y>
     <width>111</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255,255,153);


</string>
   </property>
   <property name="text">
    <string>09011</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateTC1lbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>841</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border:1px solid white;
border-radius:15px;
color:white;</string>
   </property>
   <property name="text">
    <string> ACM复位</string>
   </property>
  </widget>
  <widget class="QLabel" name="ACMStateTC2lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>400</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainNumM2lbl">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>140</y>
     <width>111</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255,255,153);
</string>
   </property>
   <property name="text">
    <string>09014</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EBStateM2lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:green;
border-left:3px solid white;
border-top:3px solid white;


</string>
   </property>
   <property name="text">
    <string>DCU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CabTC2lbl">
   <property name="geometry">
    <rect>
     <x>854</x>
     <y>140</y>
     <width>31</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TC2traingraphKeyact.bmp);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VVVFMP1Btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VVVFMP2Btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VVVFM1Btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VVVFM2Btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACMRESETTC1Btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACMRESETM1Btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACMRESETTC2Btn">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ACMRESETM2Btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <zorder>TrainNumM1lbl</zorder>
  <zorder>label</zorder>
  <zorder>DirectTC1lbl</zorder>
  <zorder>returnBtn</zorder>
  <zorder>TrainNumMP2lbl</zorder>
  <zorder>TrainNumTC2lbl</zorder>
  <zorder>TrainNumMP1lbl</zorder>
  <zorder>label_2</zorder>
  <zorder>DirectTC2lbl</zorder>
  <zorder>CabTC1lbl</zorder>
  <zorder>TrainNumTC1lbl</zorder>
  <zorder>label_3</zorder>
  <zorder>TrainNumM2lbl</zorder>
  <zorder>CabTC2lbl</zorder>
  <zorder>VVVFMP1Btn</zorder>
  <zorder>VVVFMP2Btn</zorder>
  <zorder>VVVFM1Btn</zorder>
  <zorder>VVVFM2Btn</zorder>
  <zorder>ACMRESETTC1Btn</zorder>
  <zorder>ACMRESETM1Btn</zorder>
  <zorder>ACMRESETTC2Btn</zorder>
  <zorder>ACMRESETM2Btn</zorder>
  <zorder>ACMStateM2lbl</zorder>
  <zorder>EBStateM1lbl</zorder>
  <zorder>EBStateMP2lbl</zorder>
  <zorder>ACMStateTC2lbl</zorder>
  <zorder>EBStateMP1lbl</zorder>
  <zorder>ACMStateTC1lbl</zorder>
  <zorder>ACMStateM1lbl</zorder>
  <zorder>EBStateM2lbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
